### PR TITLE
Add ModMyPi Christmas Tree Star

### PR DESCRIPTION
**Pull request by tuftii**
Next board I've added from MMP is the Christmas Star we released last year. A rather lengthy one as it uses nearly all the GPIO pins, but here it is! Let me know if anything needs adding/changing. :smile:

### CODE REVIEW

File: draft/overlay/modmypi-star.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  '9':
+  '14':
+  '20':
+  '25':
+  '30':
+  '34':
+  '39':
+pin:
+  '3':
+    name: LED Inner
```
Comment (lurch) [inline draft/overlay/modmypi-star.md]: I did a bit of poking around, and it seems this actually controls the 5 inner LEDs all at once? Perhaps "Inner LEDs" would be a better name than "LED Inner"? :man_shrugging:

Comment (tuftii) [reply draft/overlay/modmypi-star.md]: I think I was trying to match it to the GPIO Zero extension I wrote in the library which uses star.inner to control the LEDs... Happy to change it if it makes more sense reading it as Inner LEDs.

Comment (lurch) [reply draft/overlay/modmypi-star.md]: I'll leave it up to @Gadgetoid as an editor's decision :wink:

**Comment by Gadgetoid** [comment]
You're just determined to ruin the nice neat lines on the board listing page, aren't you?

Thank you, I'll get this merged!

File: draft/overlay/modmypi-star.md (hunk 1 above)
Comment (Gadgetoid) [reply draft/overlay/modmypi-star.md]: I'm going with plan C - remove the word "LED" from in front of each pin name because we know they're all LEDs.

**Comment by tuftii** [comment]
Trying my best to leave chaos and ruin in my wake :wink:

**Pull request merged**